### PR TITLE
NET-374: Change location of generated config file in broker config wizard

### PR DESCRIPTION
**Pull request by bearni95**

### CODE REVIEW

File: packages/broker/test/unit/ConfigWizard.test.ts [hunk 1]
```diff
@@ -8,6 +8,19 @@ const mockPromptMethod = (wizard: ConfigWizard, mockedPromptResult: any) => {
 }
 describe('ConfigWizard', () => {
     let wizard: ConfigWizard
+    const tmpDataDir = '/tmp/streamr-broker-test/'
```
Comment (harbu) [inline packages/broker/test/unit/ConfigWizard.test.ts]: Would be a bit more flexible to rely on `os.tmpdir()` instead to fetch temp directory (which could be different depending on operating system)

File: packages/broker/test/unit/ConfigWizard.test.ts [hunk 2]
```diff
@@ -8,6 +8,19 @@ const mockPromptMethod = (wizard: ConfigWizard, mockedPromptResult: any) => {
...
+    
+    // create a clean dir for the tests on every run
+    beforeAll(() => {
+        if (existsSync(tmpDataDir)) {
```
Comment (harbu) [inline packages/broker/test/unit/ConfigWizard.test.ts]: Seems slightly hazardous to be deleting a potentially pre-existing folder here. You might want to look at using `mkdtempSync` see: https://blog.mastykarz.nl/create-temp-directory-app-node-js/ that way you can generate a unique temporary directory so you don't have to worry about it pre-existing and only leave cleanup to `afterAll`.

**Review by harbu — CHANGES_REQUESTED** [review]
LGTM; see comments relating to test

**Comment by harbu** [comment]
Hey I get the following error when trying to run the wizard:

```
(node:18811) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open '/home/harbu/work/monorepo/packages/broker/dist/configs/wizard-config.json'
    at Object.openSync (fs.js:476:3)
    at Object.writeFileSync (fs.js:1467:35)
    at ConfigWizard.storeConfig (/home/harbu/work/monorepo/packages/broker/dist/src/ConfigWizard.js:169:14)
    at ConfigWizard.start (/home/harbu/work/monorepo/packages/broker/dist/src/ConfigWizard.js:175:44)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:18811) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:18811) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by harbu** [comment]
> Hey I get the following error when trying to run the wizard:
> 
> ```
> (node:18811) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open '/home/harbu/work/monorepo/packages/broker/dist/configs/wizard-config.json'
>     at Object.openSync (fs.js:476:3)
>     at Object.writeFileSync (fs.js:1467:35)
>     at ConfigWizard.storeConfig (/home/harbu/work/monorepo/packages/broker/dist/src/ConfigWizard.js:169:14)
>     at ConfigWizard.start (/home/harbu/work/monorepo/packages/broker/dist/src/ConfigWizard.js:175:44)
>     at processTicksAndRejections (internal/process/task_queues.js:93:5)
> (Use `node --trace-warnings ...` to show where the warning was created)
> (node:18811) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
> (node:18811) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
> ```

NVM this no longer happens, I hadn't built project properly I think.

File: packages/broker/test/unit/ConfigWizard.test.ts (hunk 1 above)
Comment (bearni95) [reply packages/broker/test/unit/ConfigWizard.test.ts]: didn't know about `os.tmpdir()`, right what was needed!

File: packages/broker/test/unit/ConfigWizard.test.ts (hunk 2 above)
Comment (bearni95) [reply packages/broker/test/unit/ConfigWizard.test.ts]: Interrupted tests were making a mess, this is a much more elegant solution

**Review by harbu — APPROVED** [review]

**Pull request merged**